README.md

```diff
@@ -110,74 +108,55 @@ Note on AIX Support:
 
 ### Configure node
 
-Once the cookbook is available in Chef Server, you need to add the `audit::default` recipe to the run-list of each node. The profiles are selected via the `node['audit']['profiles']` attribute. A complete list of the possible configuration are documented in [Supported Configurations](docs/supported_configuration.md). For example you can define the attributes in a role or environment file like this:
+Once the cookbook is available in Chef Server, you need to add the `audit::default` recipe to the run-list of each node. The profiles are selected via the `node['audit']['profiles']` attribute. A list of example configurations are documented in [Supported Configurations](docs/supported_configuration.md). Below are some other examples:
 
+```ruby
+default['audit']['reporter'] = 'chef-server-compliance'
 
-node.default['audit']['profiles'].push("path": "#{PROFILES_PATH}/mylinux-failure-success")
+# Omit this to use the latest InSpec
+default['audit']['inspec-version'] = '1.29.0'
 
-```ruby
-"audit": {
-  "reporter": "chef-server-compliance",
-  "inspec_version": "1.29.0",
-  "profiles": [
-    # profile from Chef Compliance
+default['audit']['profiles'].push(
+    # Profile from Chef Compliance
     {
-      "name": "linux",
-      "compliance": "base/linux"
+      'name': 'linux',
+      'compliance': 'base/linux'
     },
-    # profile from Chef Compliance at a particular version
+    # Profile from Chef Compliance at a particular version
     {
-      "name": "linux-baseline",
-      "compliance": "user/linux-baseline",
-      "version": "2.1.0"
+      'name': 'linux-baseline',
+      'compliance': 'user/linux-baseline',
+      'version': '2.1.0'
     },
-    # profile from supermarket
+    # Profile from Supermarket
     # note: If reporting to Compliance, the Supermarket profile needs to be uploaded to Chef Compliance first
```

Comment:
If reporting to Compliance, first upload the Supermarket profile to Chef Compliance.

Reply:
Also: Compliance is deprecated. Compliance EOL is 12/31/2018.

Reply:
Try Automate!

Reply:
I rearranged the wording. Sadly this whole doc needs a "Compliance is EOL" overhaul so I'll skip that bit.